factories/gift.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+FactoryGirl.define do
+  factory :gift do
+    user { create(:user) }
+    work { create(:work) }
+    recipient "recipient"
```

Comment:
Since you already fake the user recipient, you can leave out this field, which is for non-user recipients.